cluster/gce/config-test.sh

```diff
@@ -112,8 +112,8 @@ export LOAD_IMAGE_COMMAND
 # if KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION or KUBE_UBUNTU_INSTALL_RUNC_VERSION
 # is set to empty then we do not override the version(s) and just
 # use whatever is in the default installation of containerd package
-export UBUNTU_INSTALL_CONTAINERD_VERSION=${KUBE_UBUNTU_INSTALL_CONTAINERD_VERSION:-}
-export UBUNTU_INSTALL_RUNC_VERSION=${KUBE_UBUNTU_INSTALL_RUNC_VERSION:-}
+export UBUNTU_INSTALL_CONTAINERD_VERSION="v1.3.4"
+export UBUNTU_INSTALL_RUNC_VERSION="v1.0.0-rc10"
```

Comment:
What's the reason we hardcoded containerd and runc version?

Reply:
This PR is just using CI to test. See discussion in sig-testing on slack.